web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt

```diff
@@ -357,17 +350,30 @@ fun A(
 }
 
 @Composable
-fun Input(
-    type: InputType = InputType.Text,
-    value: String = "",
-    attrs: AttrBuilderContext<HTMLInputElement> = {}
+fun <K> Input(
+    type: InputType<K>,
+    attrs: InputAttrsBuilder<K>.() -> Unit
 ) {
     TagElement(
         elementBuilder = ElementBuilder.Input,
         applyAttrs = {
-            type(type)
-            value(value)
-            attrs()
+            val iab = InputAttrsBuilder(type)
```

Comment:
let's not try to save bytes and call it al least attreBuilder )

Reply:
fixed